### DISCUSSION TITLE
current status

### DISCUSSION THREAD

**Discussion by Ennowulff** [google/CodeCity#438] [General discussion]
Hey there! 

1. Is there any demo online? Codecity.world is working but the one and only link to https://google.codecity.world/ does not work.
2. If I follow your simple 😉 installation instructions in https://github.com/google/CodeCity/blob/master/docs/setup.md 
  a. How do I access the source code to be visualized as city?
  b. Can I refer to any github repository?
  c. What languages are supported?
  d. What do I have to do to support other programming languages?

thanks
Enno

**Comment by cpcallen** [comment]
> Is there any demo online? Codecity.world is working but the one and only link to https://google.codecity.world/ does not work.

Regrettably there is no public demo at the present time.  We are hoping someone will run one that we can link to; unfortunately we are not able to do so ourselves in an official capacity.

> How do I access the source code to be visualized as city?

As you have now realised, Google Code City is unrelated to Richard Wettel's [excellent code visualisation tools](https://wettel.github.io/codecity.html).

> What languages are supported?
> What do I have to do to support other programming languages?

Google Code City is written in and runs JavaScript (mostly ES5.1, hopefully eventually ES6) and is unlikely to support any other languages in the foreseeable future.  It might be possible to support other languages but it would probably amount to an almost total rewrite of the server and database code; only the front end code for the editors and comic strip UI would likely be reusable.

**Reply by Ennowulff** [reply]
Thanks for your replay @cpcallen 
It seems that misunderstood the idea of your Code City because I first found @wettel 's tool.
I thought both code cities are similar but indeed the intent is completely different.
Thanks and good luck 🍀
Enno
